src/lib/types/SharedConversation.ts

```diff
@@ -10,4 +10,5 @@ export interface SharedConversation extends Timestamps {
 	title: string;
 	messages: Message[];
 	preprompt?: string;
+	assistantId?: string;
```

Comment:
this is not of type `Assistant["_id"]` ?

Reply:
good catch, fixed in https://github.com/huggingface/chat-ui/pull/639/commits/41a5971f1d41e088a5a0f077c0fe9ed2b5a9ed59